src/test/groovy/io/seqera/wave/controller/ContainerInspectControllerTest.groovy

```diff
@@ -58,18 +58,18 @@ class ContainerInspectControllerTest extends Specification {
         resp.body().container.registry == 'docker.io'
         resp.body().container.imageName == 'library/busybox'
```

Comment:
Let's use `library/busybox@sha256:4be429a5fbb2e71ae7958bfa558bc637cf3a61baf40a708cb8fff532b39e52d0` so it will not change overtime